cookbooks/fb_apache/metadata.rb

```diff
@@ -4,6 +4,7 @@
 license 'BSD'
 description 'Installs/Configures Apache'
 source_url 'https://github.com/facebook/chef-cookbooks/'
+issues_url 'https://github.com/facebook/chef-cookbooks/issues'
```

Comment:
I've missed `scripts/run_foodcritic` and fixed foodcritic warning. I will rollback change.